### PR TITLE
Added "Load Testing" page to API Testing

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/load-testing.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+<img src={useBaseUrl('/img/api-testing/load-test-agents-offline.png')} alt="Alert that agents are offline" width="400"/>
+
+* Virtual users - The number of active users per agent you want to simulate. The value can be any value that is less than or equal to the max virtual users available for that agent. If more than one agent is selected, the lesser virtual users value is the max number of virtual users you can set up. If you try to set more agents than the available you will see an error message.
+
+You can set up multiple profiles for each test, from less aggressive to very aggressive, or with varying durations. The only limitations are those inherited from your infrastructure.
+
+Once you have created and saved a load test, the configuration will be saved for future use. To access a previously saved configuration, on the test page, , as a list, on the page, which you access by clicking on the load test icon from the test list.
+
+The execution of a load test will generate a real-time report that is updated every minute.
+To stop a report before it is complete, click the **Stop** button.
```
Comment (lysannep) [inline docs/api-testing/load-testing.md:72]:
```suggestion
To stop a report before it is complete, click **Stop**.
```

**Review by lysannep — DISMISSED** [review]
@lauramthomas I noticed just one typo.

**Review by LoryApiFortress — CHANGES_REQUESTED** [review]

**Comment by LoryApiFortress** [comment]
@lauramthomas please proceed with the push as per request in the Jira ticket and let me know when it is available in production. Thanks!

**Review by spider-sauce — APPROVED** [review]
👍

**Pull request merged**